Resources/views/CRUD/base_list_field.html.twig

```diff
@@ -9,7 +9,7 @@ file that was distributed with this source code.
 
 #}
 
-<td class="sonata-ba-list-field sonata-ba-list-field-{{ field_description.type }}" objectId="{{ admin.id(object) }}">
+<td class="sonata-ba-list-field sonata-ba-list-field-{{ field_description.type }}" objectId="{{ admin.id(object) }}"{% if field_description.options.align is defined %} style="text-align:{{ field_description.options.align }}"{% endif %}>
```

Comment:
no, this is the alignement of inner_cells (to override the template default one)